<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>supprimer</class>
 <widget class="QWidget" name="supprimer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background: white;
}
QLabel {
color: black;
font-weight: bold;
}



#groupBox {
border: 3px solid gray;
border-radius: 40px;
background: white;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}

QPushButton#pushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
font-weight: bold;
font-size: 13px;
padding-right: 5px;
min-width: 200px;
max-width: 200px;
min-height: 12px;
max-height: 16px;

border: 2px solid #555;
border-radius: 11px;
padding: 5px;
}

QPushButton#pushButton_3 {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
font-weight: bold;
font-size: 13px;
padding-right: 5px;
min-width: 200px;
max-width: 200px;
min-height: 12px;
max-height: 16px;

border: 2px solid #555;
border-radius: 11px;
padding: 5px;
}

QPushButton#retour {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
font-weight: bold;
font-size: 13px;
padding-right: 5px;
min-width: 90px;
max-width: 90px;
min-height: 12px;
max-height: 16px;

border: 2px solid #555;
border-radius: 11px;
padding: 5px;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}

QComboBox {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>214</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Supprimer Produit/Recette</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>91</y>
       <width>214</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Supprimer Ingr√©dient</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="retour">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
